content/src/content/jcr_root/apps/core/wcm/components/image/v2/image/README.md

```diff
@@ -62,10 +62,10 @@ The images are loaded through the `com.adobe.cq.wcm.core.components.internal.ser
 
 ```
 Author:
-/content/<project_path>/<page_path>/<component_path>/<component_name>.coreimg.<quality>.<width>.<extension>/<timestamp>.<extension>
+/content/<project_path>/<page_path>/<component_path>/<component_name>.coreimg.<quality>.<width>.<extension>/<timestamp>/<filename>.<extension>
 
 Publish:
-/content/<project_path>/<page_path>/<component_path>/<component_name>.coreimg.<quality>.<width>.<extension>
+/content/<project_path>/<page_path>/<component_path>/<component_name>.coreimg.<quality>.<width>.<extension>/<filename>.<extension>
```

Comment:
@richardhand, I just noticed that I’ve missed adding `timestamp` suffixe here. Could you please add that as well if you are planning to merge today? Or I can add it tomorrow.

Reply:
@ompandeyy I was under the impression that the timestamp isn't used on publish